jbehave-support-core/src/main/java/org/jbehavesupport/core/verification/VerificationSteps.java

```diff
@@ -0,0 +1,48 @@
+package org.jbehavesupport.core.verification;
+
+import java.util.List;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import org.assertj.core.api.SoftAssertions;
+import org.codehaus.plexus.util.StringUtils;
+import org.jbehave.core.annotations.Then;
+import org.jbehave.core.model.ExamplesTable;
+import org.jbehavesupport.core.internal.parameterconverters.ExamplesEvaluationTableConverter;
+import org.jbehavesupport.core.internal.verification.VerifierNames;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import static org.jbehavesupport.core.internal.ExampleTableConstraints.DATA;
+import static org.jbehavesupport.core.internal.ExampleTableConstraints.EXPECTED_VALUE;
+import static org.jbehavesupport.core.internal.ExampleTableConstraints.VERIFIER;
+import static org.jbehavesupport.core.internal.ExamplesTableUtil.convertTable;
+
+@Component
+@RequiredArgsConstructor
+public class VerificationSteps {
+
+    @Value("${verifier.max.assert.count:10}")
+    private int maxSoftAssertCount;
+
+    private final VerifierResolver verifierResolver;
+    private final ExamplesEvaluationTableConverter tableConverter;
+
+    @Then("following data are compared: $stringTable")
+    public void compareRows(String stringTable) {
+        List<Map<String, String>> convertedTable = convertTable((ExamplesTable) tableConverter.convertValue(stringTable, null));
+        SoftAssertions softly = new SoftAssertions();
+        convertedTable.forEach(e -> verifyRow(e, verifierResolver.getVerifierByName(VerifierNames.EQ), DATA, EXPECTED_VALUE, softly));
+        softly.assertAll();
+    }
+
+    private void verifyRow(Map<String, String> tableRow, Verifier verifier, String searchColumn1, String searchColumn2, SoftAssertions softly) {
```

Comment:
1. As verifier is always the same (EQ) I suggest to remove parameter from signature and simple autowire specific verifier bean that will be used here. Or even better - only in `resolveVerifier` method.
2. `serachColumns` are also always the same value, why putting it into signature?
3. sorry for not pointing it out earlier, I overlooked this.

Reply:
@AmyKosinova it is OK now?

Reply:
Clean like queens crown :+1: